src/cdomains/floatDomain.ml

```diff
@@ -986,10 +986,6 @@ module FloatDomTupleImpl = struct
     Option.map_default identity ""
       (mapp { fp= (fun (type a) (module F : FloatDomain with type t = a) x -> F.name () ^ ":" ^ F.show x); } x)
 
-  let hash x =
-    Option.map_default identity 0
-      (mapp { fp= (fun (type a) (module F : FloatDomain with type t = a) -> F.hash); } x)
-
```

Comment:
Is the derived one really equivalent here?

Reply:
It might have different integer values but that's beside the point. `equal` and `compare` are already being derived, so manually defining `hash` is just error-prone if things ever go out of sync.